### ISSUE TITLE
Defect report on glow/trainer.py

### ISSUE THREAD

**Issue by kelvinqin** [simonalexanderson/StyleGestures#25]
Simon,
I am thinking the following code of trainer.py may have a problem ---

            print(
                f'Loss: {loss.item():.5f}/ Validation Loss: {loss_val:.5f} '
            )

Here loss's value was updated per batch (**it was not accumulated per batch**), so here I guess the attention is to print out an average loss of the total train set, but it actually print out the loss of the last batch of each epoch,

Please let me know your thoughts, maybe I am wrong?

Have a nice day,
